src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx

```diff
@@ -344,6 +344,20 @@ function ComposerWithSuggestions(
         [],
     );
 
+    useEffect(() => {
+        const switchToCurrentReport = DeviceEventEmitter.addListener(`switchToCurrentReport_${reportID}`, ({preexistingReportID, callback}) => {
```

Comment:
NAB `switchToCurrentReport_` can be `switchToPreExistingReport_ `. Because that's what the event it is right?

Reply:
Make sense.

Reply:
I updated.